spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BindingServiceConfiguration.java

```diff
@@ -164,12 +169,23 @@ public BinderAwareChannelResolver binderAwareChannelResolver(BindingService bind
 	}
 
 	@Bean
-	@ConditionalOnProperty("spring.cloud.stream.bindings." + ERROR_CHANNEL_NAME + ".destination")
-	public SingleBindingTargetBindable<MessageChannel> errorChannelBindable(
-			@Qualifier(IntegrationContextUtils.ERROR_CHANNEL_BEAN_NAME) PublishSubscribeChannel errorChannel,
+	@ConditionalOnProperty("spring.cloud.stream.bindings." + ERROR_NAME + ".destination")
```

Comment:
@mbogoevici it might be useful to have this exposed as a bean for testing purposes, as we can then subscribe to the bridged channel directly and make assertions.